p4-16/psa/PSA.mdk

```diff
@@ -1231,8 +1264,14 @@ The PRE extern object has no constructor, and thus it cannot be
 instantiated in the user's P4 program. The architecture instantiates
 it exactly once, without requiring the user's P4 program to
 instantiate it. The PRE is made available to the `PSA_Switch` package.
-A corresponding Buffering and Queuing Engine (BQE) extern
-is defined for the egress pipeline (see [#sec-bqe]).
+
+~ TBD
+The statement "The PRE extern object has no constructor" is currently
+wrong.  We should either make it correct by changing psa.p4 somehow,
+or update the text above to match psa.p4.
+
+The same comment applies for the BQE section below.
```

Comment:
Let's update the text. We need quite a bit more work in the language/compiler to support the implicitly constructed externs. Not likely to get my wish before the end of Jan.

Reply:
Will soon add another commit to this PR that updates this text, and removed the TBD.